examples/deepzoom/main.js

```diff
@@ -13,38 +17,47 @@ $(function () {
       return geo.imageTile({
         index: index,
         size: {x: this._options.tileWidth, y: this._options.tileHeight},
+        queue: this._queue,
         url: this._options.url(source || index)
       });
     };
   };
 
-  DeepZoom.defaults = $.extend({}, geo.tileLayer.defaults, {
-    minLevel: 0,
-    maxLevel: 9,
+  // We know the size of the image we are requesting, and we want to use
+  // pixel coordinates on our map.
+  var sizeX = 103583, sizeY = 70014, tileSize = 256;
+  var defaultParams = geo.util.pixelCoordinateParams(
+        '#map', sizeX, sizeY, tileSize, tileSize);
+
+  DeepZoom.defaults = $.extend({}, geo.tileLayer.defaults, defaultParams.layer, {
```

Comment:
Would it be better to mix in the tile layer defaults inside `pixelCoordinateParams`?